examples/add_vectors_coords.py

```diff
@@ -17,20 +18,26 @@
     # create the viewer and window
     viewer = ViewerApp()
 
+    layer = viewer.add_image(data.camera(), name='photographer')
+    layer.colormap = 'gray'
+
     # sample vector coord-like data
     n = 1000
     pos = np.zeros((n, 4), dtype=np.float32)
     phi_space = np.linspace(0, 4*np.pi, n)
     radius_space = np.linspace(0, 100, n)
 
     # assign x-y position
-    pos[:, 0] = radius_space*np.cos(phi_space)
-    pos[:, 1] = radius_space*np.sin(phi_space)
+    pos[:, 0] = radius_space*np.cos(phi_space) + 256
+    pos[:, 1] = radius_space*np.sin(phi_space) + 256
 
     # assign x-y projection
-    pos[:, 2] = radius_space*np.cos(phi_space)
-    pos[:, 3] = radius_space*np.sin(phi_space)
+    pos[:, 2] = 2*radius_space*np.cos(phi_space)
+    pos[:, 3] = 2*radius_space*np.sin(phi_space)
 
     # add the vectors
-    viewer.add_vectors(pos)
+    layer = viewer.add_vectors(pos, width=0.4)
 
+    print(layer.vectors.shape)
+    print(layer._mesh_vertices.shape)
+    print(layer._mesh_triangles.shape)
```

Comment:
@sofroniewn can you remove these print statements now?

Reply:
done